eks-distro-base/Dockerfile.minimal-helpers

```diff
@@ -30,9 +30,17 @@ COPY --from=builder /tmp/packages/* .
 ARG BUILT_BUILDER_IMAGE
 FROM --platform=$BUILDPLATFORM ${BUILT_BUILDER_IMAGE} as run_validate
 
+# RUN yum install -y file
+
+# Copy scripts in every variant since we do not rebuild the base
+# every time these scripts change. This ensures whenever a variant is
+# built it has the latest scripts in the builder
+COPY scripts/ /usr/bin
+
 RUN set -x && \
     validate_libraries && \
     validate_symlinks && \
+    # find $NEWROOT -type f -exec file {} \; | egrep -i 'bash|shell' && \
```

Comment:
i am experimenting with checking for bash scripts in image without base, might be somehting worth a warning?